README.md

```diff
@@ -29,6 +30,8 @@ Examples:
 
 `EXCLUDES` is a comma-separated list of glob(3) patterns to exclude from backups. By default excludes all jar files (plugins, server files), logs folder and cache (used by i.e. PaperMC server).
 
+`LINK_LATEST` is a true/false (1/0) flag that creates a symbolic link to the latest backup.

```

Comment:
It would be really nice if it could accept the string values "TRUE" or "FALSE" in addition to/instead of. Using `${LINK_LATEST^^}` expansion has helped me previously with generalizing those input values.

Reply:
Using `^^` is a neat trick and simplifies using "TRUE" & "FALSE" greatly. I've added it and tested it, works great!